src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor_Minimal.cs

```diff
@@ -196,26 +196,39 @@ private void MinimallyQualify(INamedTypeSymbol symbol)
             // NOTE(cyrusn): If we're currently in a block of usings, then we want to collect the
             // aliases that are higher up than this block.  Using aliases declared in a block of
             // usings are not usable from within that same block.
-            var usingDirective = GetAncestorOrThis<UsingDirectiveSyntax>(startNode);
-            if (usingDirective != null)
+            while (startNode != null)
```

Comment:
>while (startNode != null) [](http://example.com/codeflow?start=12&length=25)

It would be good to add a comment explaining that with this loop we are trying to get out of immediately enclosing `using` directive, if any.

Reply:
Done in commit 2